packages/hs-react-ui/src/components/Button/Button.stories.tsx

```diff
@@ -25,4 +29,49 @@ export const Basic = () => (
 
 Basic.story = {
   name: 'Basic'
-};
\ No newline at end of file
+};
+
+// text container on figma, icons, icon yes/no and then send in icon
+const ThemedContainer = styled.button`
```

Comment:
Remember, you can import a base Container and then instead of `styled.button` you can just do `styled(PrimaryButtonContainer)` - and only include the styles you want to change